src/libraries/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -1079,16 +1079,14 @@ private sealed class WindowsConsoleStream : ConsoleStream
             // is Encoding.Unicode implying 2 bytes per character:
             private const int BytesPerWChar = 2;
 
-            private readonly bool _isPipe; // When reading from pipes, we need to properly handle EOF cases.
```

Comment:
So we were computing this but it was actually entirely unused? Wonderful :-)

Reply:
I wondered why any of the analyzers did not flag this for unused fields? Hmm, interesting.

Reply:
It's passed to a method but that method doesn't use it.

Reply:
yeah, I saw that after my first reply. But shouldn't the analyzer issue a warning for unused method parameters? It should, right? I have seen this kind of warning as well.